src/Jellyfin.Plugin.ListenBrainz/Configuration/configurationPage.html

```diff
@@ -405,6 +414,51 @@ <h3>General Config</h3>
             return inputValue.trim();
         }
 
+        function buildAllowedLibrariesList() {
```

Comment:
Allow configuring all libraries instead of only music libraries.
By default, all music libraries are included, the other types are excluded.